User Profile - More REST spec, tests, API docs (#84597)
elastic/elasticsearch: x-pack/docs/en/rest-api/security/enable-user-profile.asciidoc

[role="xpack"]
[[security-api-enable-user-profile]]
=== Enable user profile API
++++
<titleabbrev>Enable user profile</titleabbrev>
++++

beta::[]

Enables a user profile so it's visible in
<<security-api-search-user-profile,user profile searches>>.

[[security-api-enable-user-profile-request]]
==== {api-request-title}

`POST /_security/profile/<uid>/_enable`

`PUT /_security/profile/<uid>/_enable`

[[security-api-enable-user-profile-prereqs]]
==== {api-prereq-title}

To use this API, you must have the `manage_user_profile` cluster privilege.

[[security-api-enable-user-profile-desc]]
==== {api-description-title}

When you <<security-api-activiate-user-profile,activate a user profile>>, it's

> automatically enabled and visible in user profile searches.  If you later

Reviewer: ```suggestion
automatically enabled and visible in user profile searches. If you later
```

> <<security-api-disable-user-profile,disable a user profile>>, you can use the
> enable user profile API to make it visible these searches again.

Reviewer: ```suggestion
<<security-api-disable-user-profile,disable the user profile>>, you can use the
enable user profile API to make the profile visible in these searches again.
```

[role="xpack"]
[[security-api-enable-user-profile]]
=== Enable user profile API

beta::[]

++++
<titleabbrev>Enable user profile</titleabbrev>

> ++++

Author: I thought `beta::[]` should be before the title. Apparently that is wrong. In this case, I'll need to change it in a few other pages related to user profile.
